### PR TITLE
[14.0][MIG] account_invoice_check_picking_date: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0

Please @carlosdauden and @sergio-teruel can you review it?

@Tecnativa TT40069

### CODE REVIEW

**Review by sergio-teruel — COMMENTED** [review]
Code review

**Review by sergio-teruel — APPROVED** [review]

File: account_invoice_check_picking_date/README.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+============
+
+#. You can define the config_parameter "account_invoice_check_picking_date.invoice_types" with a list
+of invoice type separated by comma. Ex. "in_invoice,in_refund"
+
+Usage
+=====
+
+#. Go to *Purchases > Purchase Orders > Create*.
+#. Choose a customer and add a stockable product.
```
Comment (petrus-v) [inline account_invoice_check_picking_date/README.rst]:
```suggestion
#. Choose a supplier and add a stockable product.
```

**Review by petrus-v — COMMENTED** [review]
Could be nice to add some unit test !

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**